sample/src/main/java/com/airbnb/deeplinkdispatch/sample/SecondActivity.java

```diff
@@ -2,14 +2,40 @@
 
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
+import android.text.TextUtils;
+import android.util.Log;
+import android.widget.Toast;
 
 import com.airbnb.deeplinkdispatch.DeepLink;
 
-@DeepLink("dld://example.com/deepLink/{id}")
+// You can have multiple path parameters in the URI - both must be present
+@DeepLink("http://example.com/deepLink/{id}/{name}")
 public class SecondActivity extends AppCompatActivity {
 
+    private String TAG = SecondActivity.class.getSimpleName();
+
     @Override protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        if (getIntent().getBooleanExtra(DeepLink.IS_DEEP_LINK, false)) {
+            Bundle parameters = getIntent().getExtras();
+            Log.d(TAG, "Deeplink params: " + parameters);
+
+            String idString = parameters.getString("id");
+            String name = parameters.getString("name");
+            if(!TextUtils.isEmpty(idString))
+                showToast("class id== " + idString + " and name==" + name);
+            else{
+                showToast("no id in the deeplink");
+            }
+        }else{
```

Comment:
formatting is a little off in this file. Can you please reformat it to use 2 spaces indentation? also spaces before curly braces and `else` statement